.prospector.yml

```diff
@@ -24,6 +25,7 @@ pylint:
 pycodestyle:
   disable:
     - E731 # do not assign a lambda expression, use a def
+    - N804 # first argument of a classmethod should be named 'cls' (I use 'mcls' for metaclasses)
```

Comment:
Metaclasses are classes too, so this choice isn't justified. ( ͡• ͜ʖ ͡• )

Reply:
Metaclasses receive a completely different set of arguments, and namiing it `mcls` makes it clearer what is what.